### ISSUE TITLE
Could you tag your latest release?

### ISSUE THREAD

**Issue by JesseTG** [sttz/trimmer#7]
When you get a chance, would you mind tagging your latest release with the appropriate version number so it'll appear on [OpenUPM](https://openupm.com/packages/ch.sttz.trimmer/)? Thank you!

**Comment by sttz** [comment]
Sorry, I assumed my client pushed the tags. Pushed them now.

Adding `OptionInclusion.Build` in 1.1.3 caused me some headaches. I added it because e.g. build-only options would always get `OptionInclusion.Remove`, irrespective of wether the current platform is part of their `SupportedPlatforms`.

But adding it broke some checks I did, where I just compared `inclusion == OptionInclusion.Remove` or `inclusion != OptionInclusion.Remove` and which didn't work anymore when inclusion was `Build`. I now think it's better to always check for the presence of the relevant flag (e.g. `inclusion.HasFlag(OptionInclusion.Feature)`) instead of comparing to `Remove`.

Don't know if you're affected by this as well, just something to look out for.
